packages/search-ui/src/actions/a11yNotify.ts

```diff
@@ -7,7 +7,7 @@ import { announceToScreenReader } from "../A11yNotifications";
  * @param {string} messageFunc - key of a message function in `a11yNotificationMessages`
  * @param {object} [messageArgs] - arguments to pass to the message function, if any
  */
-export default function a11yNotify(messageFunc, messageArgs) {
+export default function a11yNotify(messageFunc: string, messageArgs?: any): void {
```

Comment:
I don't know why I skipped all function return types before 😄 
It seems that in this place and in several others, function return types are not needed because TS can infer that from the code. What do you think about removing them?

Reply:
soo my stance here is being explicit is good as the underlying code may change (the return type may change based on a refactor) and wont be obvious that it broke until you look at the references using that method.